packages/dropdown/stories/it-dropdown.mdx

```diff
@@ -16,7 +16,7 @@ Il componente dropdown, chiamato anche menu a tendina, offre una navigazione vel
 
 ### Alternative a questo componente
 
-Select, Autocomplete e [Radio button](/docs/componenti-form-radio--documentazione):
+[Select](/docs/componenti-form-select--documentazione), [Autocomplete](/docs/componenti-form-autocomplete--documentazione) e [Radio button](/docs/componenti-form-radio--documentazione):
```

Comment:
In altri componenti ci era stato dato come feedback di non mettere link non funzionanti finché non sono rilasciati quindi aspetterei a fare questa modifica per ora.